src/main/java/me/jellysquid/mods/sodium/client/gui/SodiumGameOptions.java

```diff
@@ -139,22 +144,38 @@ public String getLocalizedName() {
             .excludeFieldsWithModifiers(Modifier.PRIVATE)
             .create();
 
-    public static SodiumGameOptions load(File file) {
+    public static SodiumGameOptions load(Path path) {
         SodiumGameOptions config;
 
-        if (file.exists()) {
-            try (FileReader reader = new FileReader(file)) {
+        if (Files.exists(path)) {
+            try (InputStream is = Files.newInputStream(path);
+                 InputStreamReader isr = new InputStreamReader(is);
+                 BufferedReader reader = new BufferedReader(isr)) {
                 config = gson.fromJson(reader, SodiumGameOptions.class);
-            } catch (IOException e) {
-                throw new RuntimeException("Could not parse config", e);
+            } catch (Exception e) {
+                LOGGER.error("Failed to parse options file!", e);
+                LocalDateTime now = LocalDateTime.now();
+                String backupName = path.getFileName().toString();
+                backupName = backupName.substring(0, backupName.lastIndexOf('.') - 1);
+                backupName += "-" + now.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME) + ".json";
+                Path backupPath = path.resolveSibling(backupName);
+                LOGGER.info("Backing up config to \"{}\"...", backupPath.toString());
+                try {
+                    Files.move(path, backupPath, StandardCopyOption.REPLACE_EXISTING);
+                } catch (IOException be) {
+                    LOGGER.error("Failed to back up config!", be);
+                }
+                LOGGER.info("Loading default values");
+                config = new SodiumGameOptions();
             }
 
             config.sanitize();
         } else {
+            LOGGER.info("Could not find options file, loading default values");
             config = new SodiumGameOptions();
         }
 
-        config.file = file;
+        config.path = path;
         config.writeChanges();
```

Comment:
I have mentioned this in `#sodium` before. Perhaps we should only write changes once when creating the config file for this first time.
```suggestion
        if (!File.exists(path)) {
                config.writeChanges();
        }
```